homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
The dependency is not built and published in a CI

Reply:
We don't actually _know_ that though. huawei-lte-api and stringcase don't seem to be published from a _public_ CI it seems, right, downgraded to TODO and added a note in aafc6850804490a07a37faaaf8235e08f2e62b4b. Also with that assumption, added a clarifying PR for the docs in https://github.com/home-assistant/developers.home-assistant/pull/2656

Also, the item about building/publishing in a CI is a "should", not "must", in the rule documentation. There are two "must"s and two "should"s in the doc. Given the doc is all about the quality scale rule, I feel there should be a difference between "must" and "should". Or perhaps we can mark the item as done, but add a comment when a "should" item is not addressed?

Reply:
Publishing in a public CI is a must as with a project this size, that is the least we can do to make our dependencies more transparent

Reply:
In that case, it sounds that the documentation at https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/dependency-transparency/ needs to be updated to say "must" for all items, not "should" for some.